oprf-client/src/lib.rs

```diff
@@ -202,14 +194,18 @@ pub struct NullifierArgs {
     pub merkle_membership: MerkleMembership,
     /// The original OPRF query (RP ID, action, nonce, timestamp, etc.).
     pub query: OprfQuery,
-    /// Groth16 material (proving key and matrices) used for query and nullifier proofs.
-    pub groth16_material: Groth16Material,
+    /// Groth16 material (proving key and matrices) used for query proof.
+    pub query_material: Groth16Material,
```

Comment:
This should probably be an `Arc<Groth16Material>` since it is quite big and we clone into here.